### PR TITLE
Small update to exercise command

### PR DESCRIPTION
**Pull request by andresbyun**
`npx vitest auth.github` won't run because it won't find files under that name since the relevant file is `auth.$provider.callback.test.ts`.

What it is right now:
image

Error message:
```
> npx vitest auth.github

 DEV  v0.34.6 /repos/epicweb-dev/full-stack-testing/playground

filter:  auth.github
include: ./app/**/*.test.{ts,tsx}
exclude:  **/node_modules/**, **/dist/**, **/cypress/**, **/.{idea,git,cache,output,temp}/**, **/{karma,rollup,webpack,vite,vitest,jest,ava,babel,nyc,cypress,tsup,build}.config.*
watch exclude:  **/node_modules/**, **/dist/**

No test files found, exiting with code 1
```

### CODE REVIEW

File: exercises/08.http-mocking/01.problem.start-server/README.mdx [hunk 1]
```diff
@@ -26,5 +26,5 @@ you that does the conversion for you. You'll find it at the bottom of the test
 file. It's called `convertSetCookieToCookie`.
 
 ```sh nonumber
-npx vitest auth.github
+npx vitest auth.$provider
```
Comment (kentcdodds) [inline exercises/08.http-mocking/01.problem.start-server/README.mdx:29]: Terminals don't like the dollar, so I think this should work. Let me know.

```suggestion
npx vitest auth.\$provider
```

**Review by kentcdodds — COMMENTED** [review]
Thanks! One question.

**Comment by andresbyun** [comment]
@kentcdodds Updated it and tested that it works

**Review by kentcdodds — APPROVED** [review]
Thanks!

**Pull request merged**